vms/platformvm/txs/fee/complexity.go

```diff
@@ -103,42 +112,32 @@ var (
 			wrappers.IntLen + // genesis length
 			wrappers.IntLen + // subnetAuth typeID
 			wrappers.IntLen, // subnetAuthCredential typeID
-		gas.DBRead:  1,
-		gas.DBWrite: 1,
-		gas.Compute: 0,
+		gas.DBRead:  3, // get subnet auth + check for subnet transformation + check for subnet conversion
+		gas.DBWrite: 1, // put chain
```

Comment:
i tried following `CreateChainTx` and oddly enough, i don't see that it [writes the chain](https://github.com/ava-labs/avalanchego/blob/92f0f596f154233b9d9a1e3288b2b32a964c7a16/vms/platformvm/txs/executor/standard_tx_executor.go#L235) to the DB, only to memory. Can you point out where it writes it to the DB?

Reply:
https://github.com/ava-labs/avalanchego/blob/master/vms/platformvm/state/state.go#L3041